src/controllers/bespokeOrStandardRules.controller.js

```diff
@@ -28,9 +31,12 @@ module.exports = class BespokeOrStandardRulesController extends BaseController {
       return this.doGet(request, h, errors)
     }
 
-    const isBespoke = request.payload['permit-type'] === 'bespoke'
+    const context = await RecoveryService.createApplicationContext(h)
+    // Save the permit type in the Data store
+    const permitType = request.payload['permit-type']
+    await DataStore.save(context, { permitType })
 
-    if (isBespoke) {
+    if (permitType === BESPOKE) {
```

Comment:
We need to amend the end of the triage process as well so that it goes to the new bespoke task list rather than (as currently) to the apply offline page.  This is line 91 in the routes.js file.

Reply:
Done